root.tf

```diff
@@ -290,7 +295,6 @@ module "checksum_sqs_queue" {
   common_tags              = local.common_tags
   project                  = var.project
   function                 = "checksum"
```

Comment:
Is the function variable needed here anymore? Should it be "checksum"?

Reply:
Yeah I think so, it's used to create the queue name
 `sqs_name       = "${var.project}-${var.function}-${local.environment}"`
We still need the checksum queue because that's where the download files lambda is sending its messages.